src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -5216,6 +5241,7 @@ private bool IsConstructorAccessible(MethodSymbol constructor, ref CompoundUseSi
             NamedTypeSymbol type,
             AnalyzedArguments analyzedArguments,
             BindingDiagnosticBag diagnostics,
+            out bool overloadResolutionSucceeded,
```

Comment:
Trying to understand this parameter a little bit. Looks like you're using in the string binding to say 'if overload resolution failed' then try again with a different set of params. In that case wont you get a `BoundBadExpression` with `LookupResultKind.OverloadResolutionFailure`?

Reply:
This is a good suggestion, I think that should totally work.